src/webui/api/torrentscontroller.cpp

```diff
@@ -835,3 +856,91 @@ void TorrentsController::removeCategoriesAction()
     for (const QString &category : categories)
         BitTorrent::Session::instance()->removeCategory(category);
 }
+
+QVariantList TorrentsController::getStickyTrackers(const BitTorrent::TorrentHandle* torrent)
```

Comment:
const BitTorrent::TorrentHandle *torrent

Reply:
Does this function depend on TorrentsController? If not, it would be better to have it in an anonymous namespace instead.

Reply:
Will move it to an anonymous namespace